packages/cinterop/build.gradle.kts

```diff
@@ -137,7 +154,7 @@ kotlin {
             // ... and def file does not support using environment variables
             // https://github.com/JetBrains/kotlin-native/issues/3631
             // so resolving paths through gradle
-            kotlinOptions.freeCompilerArgs += nativeLibraryIncludes
+            kotlinOptions.freeCompilerArgs += nativeLibraryIncludesMacosUniversal
```

Comment:
Does these _universal_ builds eliminate the option to use the monorepo CMAKE recipes?